### PR TITLE
In processing profile screen, login and sign up.

### PR DESCRIPTION
**Pull request by MiqayelChaloyan**
### Description for PR

I finished the logic of sign up and login pages.

### CODE REVIEW

File: components/Header/Header.jsx [hunk 1]
```diff
@@ -16,61 +15,65 @@ const Header = ({
...
   headerTextColor,
 }) => {
-  const {setLoggedIn} = useContext(GlobalDataContext);
-
-  const handleLogout = async key => {
-    await removeItem(key);
-    setLoggedIn(false);
-    Alert.alert('Logout successful');
+  const [isModalVisible, setModalVisible] = useState(false);
+  const handleLogout = () => {
```
Comment (Varuj2002) [inline components/Header/Header.jsx:19]: const handleLogout = () => setModalVisible(true)

File: screens/UsersScreen/style.js [hunk 2]
```diff
@@ -32,16 +32,17 @@ const styles = StyleSheet.create({
...
     borderRadius: 50,
-    top: 6,
+    top: 4,
     right: 15,
-    // resizeMode: 'contain',
+    borderWidth: 0.5,
+    borderColor: theme.colors.primary_green,
   },
 
   //RenderImagePairs.jsx
```
Comment (Varuj2002) [inline screens/UsersScreen/style.js:48]: ?

File: contexts/context.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import {createContext, useState} from 'react';
 import settings from '../data/settings';
+// import {user} from '../data/user';
```
[the remark this answers is not in the document]
Comment (MiqayelChaloyan) [reply contexts/context.js:3]: I will delete this.

File: components/Header/Header.jsx (hunk 1 above)
Comment (MiqayelChaloyan) [reply components/Header/Header.jsx:19]: I will fix this.

File: screens/UsersScreen/style.js (hunk 2 above)
Comment (MiqayelChaloyan) [reply screens/UsersScreen/style.js:48]: The image didn't look nice back in resize mode.

**Pull request merged**